Core/Services/Auth/ExternalLoginService.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Core.Interfaces.Auth;
+using Core.ServiceResponses;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Storage.Identity;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace Core.Services.Auth
+{
+    class ExternalLoginService : AuthServicesProvider, IExternalLoginService
+    {
+        private readonly IAdditionalAuthMetods _additionalAuthMetods;
+        public ExternalLoginService(UserManager<User> _userManager, SignInManager<User> _signIn, IConfiguration _config, IJwtGenerator _jwtGenerator, IAdditionalAuthMetods add) 
+            : base(_userManager, _signIn, _config, _jwtGenerator)
+        {
+            _additionalAuthMetods = add;
```

Comment:
What the name 'add' means? Give it a proper name

Reply:
Done